client/public/index.html

```diff
@@ -96,6 +96,17 @@
   </head>
 
   <body>
+    <script>
+      document.body.addEventListener(
+        "load",
+        (e) => {
+          if (e.target.classList.contains("interactive")) {
+            e.target.setAttribute("data-readystate", "complete");
+          }
+        },
+        { capture: true }
+      );
```

Comment:
1. Could we also use a class like `loaded` for this and check for `iexFrame.classList.contains('loaded')`?
2. We probably need to merge these two `<script>` tags, to avoid managing two CSP values.